src/llmq/blockprocessor.cpp

```diff
@@ -317,8 +316,8 @@ bool CQuorumBlockProcessor::UndoBlock(const CBlock& block, gsl::not_null<const C
         return false;
     }
 
-    for (auto& p : qcs) {
-        auto& qc = p.second;
+    for (auto& [_, qc2] : qcs) {
+        auto& qc = qc2; // cannot capture structured binding into lambda
```

Comment:
nit: could do 22e96557f50b8696a936924e4ec65f98d62e9 instead

Reply:
Huh; interesting, but I think it's fine as is